### PR TITLE
fix(workflow): Fix Alert Rule List condition padding

### PR DESCRIPTION
**Pull request by billyvg**
Content was overflowing because of old CSS.

Old:
image

New:
image

### CODE REVIEW

**Comment by billyvg** [comment]
I like how @joshuarli is a suggested reviewer 
image

**Comment by joshuarli** [comment]
> I like how @joshuarli is a suggested reviewer
> image

lolol

File: src/sentry/static/sentry/app/views/settings/projectAlerts/ruleRow.tsx [hunk 1]
```diff
@@ -162,15 +162,14 @@ const RuleName = styled(Link)`
...
-  justify-content: space-between;
-  height: 100%;
+const Listing = styled('div')`
+  display: grid;
+  grid-gap: ${space(1)};
 `;
 
-// For tests
-const Actions = styled('div')<HasBorderProp>`
+const Conditions = styled(Listing)``;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/projectAlerts/ruleRow.tsx]: are you sure about it? I would replace Conditions with Listing

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/projectAlerts/ruleRow.tsx]: Updated

**Review by markstory — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**